### DISCUSSION TITLE
How do you hide admin options in the editwarp GUI from default players?

### DISCUSSION THREAD

**Discussion by tylerking** [Zrips/CMI#7234] [Q&A]
I'd like my players to have access to the /editwarp GUI so they can update their warp display name & item but its also shows the options to require-permission, show without permission, slot, page, and group. This is too much control for a default player. How do I disable these from showing up for non-admins?

**Comment by mrfloris** [comment]
Oof, I wasn't aware that when these were negated for the default group that they'd still show if they have permission to edit the title and item.  I wouldn't mind if that is indeed the case if that gets tidied up a bit more.

**Reply by tylerking** [reply]
Yeah it definitely didn't seem intentional and makes it easy for players to mess up their warps.
